phaseshifts/io/__init__.py

```
@@ -0,0 +1,43 @@
+
+    def write(self, model, path):
+        raise NotImplementedError
+
+    def to_model(self, data):
+        raise NotImplementedError
+
+    def from_model(self, model):
+        raise NotImplementedError
+
+
+from .model import (  # noqa: E402  # isort:skip
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)